puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -36,4 +52,23 @@
       }
     }
   }
+
+  Postgresql_psql {
+    cwd => '/',
+  }
+
+  include ::postgresql::client
+  include ::postgresql::server
+
+  Class['postgresql::server'] -> Package['postgresql-dev']
+
+  # Simple known user/pass that will allow Jenkins to create its
+  # own databases when required
+  postgresql::server::role { 'foreman':
+    password_hash => postgresql_password('foreman', 'foreman'),
+    superuser     => true,
+    login         => true,
+    require       => Service['postgresql'],
```

Comment:
I believe this one is redundant since the role definition already requires this. I know it was there before, but might as well fix it up.

Reply:
Without this, I would always get an error with the role trying to create itself before Postgres was running.